sorting/Nearly Sorted Algorithm.c

```diff
@@ -0,0 +1,31 @@
+#include<stdio.h>
+void sort(int a[],int n,int k)
+{
+    int key,j;
+    for(int i=1;i<n;i++)
+    {
+        key=a[i];
+        j=i-1;
+        while(j>=0 && a[j]>key)
+        {
+            a[j+1]=a[j];
+            j=j-1;
+        }
+        a[j+1]=key;
+    }
+}
```

Comment:
what is the purpose of taking ``k`` as input if it isn't used anywhere?
``sort`` is performing normal insertion sort, & we already have an implementation of insertion sort. Closing PR.